test/suites/migration.sh

```diff
@@ -222,6 +222,60 @@ migration() {
     lxc storage unset "lxdtest-$(basename "${LXD_DIR}")" zfs.clone_copy
   fi
 
+  lxc_remote init testimage l1:c1
+  lxc_remote copy l1:c1 l2:c2
+  lxc_remote copy l1:c1 l2:c2 --refresh
+
+  lxc_remote start l1:c1 l2:c2
+
+  # Make sure the testfile doesn't exist
+  ! lxc file pull l1:c1 -- /root/testfile1
+  ! lxc file pull l2:c2 -- /root/testfile1
+
+  #lxc_remote start l1:c1 l2:c2
+
+  # Containers may not be running when refreshing
+  ! lxc_remote copy l1:c1 l2:c2 --refresh
+
+  # Create test file in c1
+  echo test | lxc_remote file push - l1:c1/root/testfile1
+
+  lxc_remote stop l1:c1 l2:c2
```

Comment:
All `stop` and `restart` calls in the testsuite must use `--force` or we'll run into problems.